src/freenas/etc/rc.conf.local

```diff
@@ -576,15 +590,17 @@ _gen_conf()
 	while read -r nonroot mountdport statdport lockdport; do
 		if [ "${nonroot}" = 1 ]; then
 			mountdflags="${mountdflags}n"
+                        statdflags="${statdflags}n"
+                        lockdflags="${lockdflags}n"
 		fi
```

Comment:
Thanks for correcting me here.
I checked the code and analysed that I don't need parameter 'n' in case of statd and lockd, So I'll just delete these two lines.